src/LGPL-2.0-only.xml

```diff
@@ -157,6 +157,7 @@
       ordinary General Public License rather than by this special one.
     </p>
     <p>
+      GNU LIBRARY GENERAL PUBLIC LICENSE
```

Comment:
Can you wrap it to <p></p>?  The same in 5 other files.

Reply:
Not sure what you mean.

Before your change, there was:
```
<p>
   TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
 </p>
```
That is one paragraph.

After your change, there is:
```
<p>
   GNU LIBRARY GENERAL PUBLIC LICENSE
   TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
 </p>
```
That is still one paragraph. And https://spdx.org/licenses/preview/LGPL-2.0+ will render it as one paragraph:
```
GNU LIBRARY GENERAL PUBLIC LICENSE TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION 
AND MODIFICATION
```

I am asking for change:
```
<p>
   GNU LIBRARY GENERAL PUBLIC LICENSE
</p>
</p>
   TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
 </p>
```

or 

```
<p>
   GNU LIBRARY GENERAL PUBLIC LICENSE
   <br/>TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
 </p>
```

Reply:
FYI - appreciate the attention to detail @xsuchy and I think my commits may have perpetuated the lack of line break (personally, I prefer the <br/> for this case :) - but if the tests pass, I'm going to merge this. we can see how it looks and do any formatting updates later, if need be.